testsuite/CMakeLists.txt

```diff
@@ -36,9 +38,9 @@ add_subdirectory( musictests )
 add_subdirectory( cpptests )
 
 install( DIRECTORY ${TESTSUBDIRS}
-    DESTINATION ${CMAKE_INSTALL_DOCDIR}
+    DESTINATION ${CMAKE_INSTALL_DATADIR}/testsuite
     )
 
 install( PROGRAMS do_tests.sh junit_xml.sh run_test.sh summarize_tests.py
```

Comment:
``install PROGRAMS`` causes the file to get mode +x (https://cmake.org/cmake/help/latest/command/install.html#programs). We should use ``install FILES`` for the non-executable files.

Reply:
But these are all scripts, aren't they?

Reply:
Only ``do_tests.sh`` is +x, the rest contains the lines
```sh
# This file only contains helper functions for the NEST testsuite.
# It is meant to be sourced from do_tests.sh rather than being run
# directly.
```
and ``summarize_tests.py`` does not contain a shebang.

Reply:
Fixed by ad0be29.